docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -710,6 +818,65 @@ function layout_builder_custom_preprocess_media(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_views_view_unformatted().
+ */
+function layout_builder_custom_preprocess_views_view_unformatted(&$variables) {
+  // @todo Figure out how to appropriately scope this.
+  /** @var \Drupal\views\ViewExecutable $view */
+  $view = $variables['view'];
+  $variables['row_item_classes'] = [];
+
+  $row_styles = $view->display_handler->getOption('row_styles');
+  if (isset($row_styles) && !empty($variables['rows'])) {
+
+    // Convert single selection to an array for consistent processing.
+    if (!is_array($row_styles)) {
+      $row_styles = [$row_styles];
+    }
+
+    $default_classes = [
+      'card--list',
+      'card--person-teaser',
```

Comment:
Should this be more general for non-person lists?

Reply:
I think we need to generalize the `card--person-teaser` styles so that we don't have this issue.   Especially if we are using these styles for the new event content type.   I can make a follow-up issue detailing the work that needs to be done.